src/main/java/bot/Storage.java

```diff
@@ -0,0 +1,76 @@
+package bot;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import tasks.Task;
+
+/**
+ * Handles saving and loading bot data
+ */
+public class Storage {
+
+    public static final String SAVE_FILE_PATH = "data/save.txt";
+
+    private static void write(List<String> texts) {
```

Comment:
I would recommend a comment above this function as its seems to be quite a key function for this class. Perhaps just a simple description? I had a similar logic in my own work and I had to update my write-alike function several times as I keep updating the input format.

So perhaps it would be better to have a function descriptor for it?